aggregators/omnibox-is-themed/index.js

```diff
@@ -19,7 +19,7 @@
 
 const Aggregate = require('../aggregate');
 
-class AddToHomescreen extends Aggregate {
+class OmniboxThemeColor extends Aggregate {
```

Comment:
https://github.com/GoogleChrome/lighthouse/blob/1e1df34b8343171bb628983a93100c9fd45360f9/aggregators/will-get-add-to-homescreen-prompt/index.js

^ will need to update here?